motionplan/nearestNeighbor_test.go

```diff
@@ -10,32 +10,32 @@ import (
 )
 
 func TestNearestNeighbor(t *testing.T) {
-	nm := &neighborManager{nCPU: 2}
+	nm := &neighborManager{nCPU: 2, parallelNeighbors: 1000}
```

Comment:
I figured being explicit about `parallelNeighbors` would be better. Such that if the default were changed in the future to a larger number, this test would still use the parallel code for the second `nearestNeighbor` call.